JS/index.js

```diff
@@ -20,9 +20,217 @@ function contactSection() {
 }
 
 window.addEventListener('DOMContentLoaded', () => {
-  document.querySelector('.hamburger > img').addEventListener('click', displayContent);
+  document
+    .querySelector('.hamburger > img')
+    .addEventListener('click', displayContent);
   document.querySelector('.close').addEventListener('click', hideMenu);
-  document.querySelector('.portfolio').addEventListener('click', portfolioSection);
+  document
+    .querySelector('.portfolio')
+    .addEventListener('click', portfolioSection);
   document.querySelector('.about').addEventListener('click', aboutSection);
   document.querySelector('.contact').addEventListener('click', contactSection);
 });
+
+// Pop up window
+
+const projects = [
+  {
+    // First Project
+    name: 'Profesional Art Printing Data',
+    discription:
+      'First A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry`s standard',
+    discriptionDesktop:
+      'Firsst Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry`s standard dummy text ever since the 1500s,',
+    languages: ['html', 'bootstrap', 'Ruby'],
+    reference: [
```

Comment:
- [Optional] Good job so far, I noticed that when I click on the popup it shows as intended but when I scroll downwards of upwards, the main page scrolls, and when I close the popup I see that I am on the contact section, meaning the scrolling took me away from the focus which us the project section. You can fix this by adding a CSS property to the main page to stop the scrolling and it removes the CSS when you close the popup.